seeds/test/UserSeed.js

```diff
@@ -0,0 +1,8 @@
+module.exports = [
```

Comment:
so it is automatically imported over mongoose or seedMongoose? If thats the case, I do not get an error when running the tests.